tools/types/src/server.ts

```diff
@@ -22,28 +23,20 @@ export enum SortOrder {
 }
 
 export type OrderFilter = {
-  signerWallet?: string
-  signerMinAmount?: string
-  signerMaxAmount?: string
-  signerTokens?: string[]
-  senderWallet?: string
-  senderMinAmount?: string
+  orderNonce?: string
   senderMaxAmount?: string
+  senderMinAmount?: string
   senderTokens?: string[]
+  senderWallet?: string
+  signerMaxAmount?: string
+  signerMinAmount?: string
+  signerTokens?: string[]
+  signerWallet?: string
   sortField?: SortField
   sortOrder?: SortOrder
-  offset: number
+  tokenIds?: string[]
```

Comment:
Probably want to split this into `signerIds` and `senderIds` as is done with the signer and sender tokens.

Reply:
done